.github/workflows/test-elixir.yaml

```diff
@@ -97,3 +97,65 @@ jobs:
       # Step: Execute the tests.
       - name: Run tests
         run: mix test
+
+  test_macos:
+    name: macos-latest test
+    runs-on: macos-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+
+      - name: Install Erlang/Elixir
+        run: |
+          brew install elixir
+          mix local.hex --force
```

Comment:
Is this the only difference between Mac & the others? Would you be up for using an `if` statement like we do here instead of copying & pasting the rest of the block? https://github.com/PRQL/prql/blob/ff7ac7a880e3f18c7bec4b4e78aed0a7ae2b52fa/.github/workflows/test-rust.yaml#L28

Reply:
this workflow installs the elixir using a different method. other version with a matrix uses a strategy which can use multiple elixir versions

I used this 
workflow https://github.com/rusterlium/rustler/blob/80f6c4b926ce15dea4fac5389a67decdc9732ba8/.github/workflows/main.yml#L135

Reply:
I adjusted in another commit